doc/source/release-notes/Percona-XtraDB-Cluster-5.6.29-25.15.rst

```diff
@@ -0,0 +1,78 @@
+.. rn:: 5.6.29-25.15
+
+===================================
+Percona XtraDB Cluster 5.6.29-25.15 
+===================================
+
+Percona is glad to announce the release of
+|Percona XtraDB Cluster| 5.6.29-25.19 on May 20, 2016.
+Binaries are available from the
+`downloads section <http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/release-5.6.29-25.15/>`_
+or from our :ref:`software repositories <installation>`.
+
+Percona XtraDB Cluster 5.6.29-25.15 is now the current release,
+based on the following:
+
+* `Percona Server 5.6.29-76.2 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.29-76.2.html>`_
+
+* `Galera Replicator 3.15 <https://github.com/codership/galera/issues?q=milestone%3A25.3.15>`_
+
+* `Codership wsrep API 25.15 <https://github.com/codership/mysql-wsrep/issues?q=milestone%3A5.6.29-25.15>`_
+
+All Percona software is open-source and free.
+
+For more information about relevant Codership releases, see `this announcement <http://galeracluster.com/2016/03/announcing-galera-cluster-5-5-48-and-5-6-29-with-galera-3-15/>`_.
+
+Bugs Fixed
+==========
+
+:jira:`PXC-329`: Node eviction in the middle of SST now causes the node
+ to shut down properly.
+
+:jira:`PXC-473`: After an error during node startup,
+ the state is now marked *unsafe* only if SST is required.
+
+:jira:`PXC-474`: Fixed data inconsistency during multi-insert auto-increment
+ workload on async master with ``binlog-format=STATEMENT``
+ when a node begins async slave with ``wsrep_auto_increment_control=ON``.
+
+:jira:`PXC-497`: Fixed crash when a prepare statement is aborted
+ (due to conflict with applier) and then replayed.
+
+:jira:`PXC-500`: Removed a special case condition in ``wsrep_recover()``
+ that whould not happen under normal conditions.
+
+:jira:`PXC-502`: Percona XtraDB Cluster no longer fails during SST,
+ if a node reserves a very large amount of memory for InnoDB buffer pool.
+
+:jira:`PXC-507`: If the value of ``wsrep_cluster_address`` is not valid,
+ trying to create a slave thread will now generate a warning
+ instead of an error, and the thread will not be created.
+
+:jira:`PXC-508`: Fixed error with LDI when loading data
+ into a multi-partitioned table.
+
+:jira:`PXC-509`: The ``wsrep_node_name`` variable now defaults to host name.
+
+:jira:`PXC-516`: Starting ``mysqld`` with unknown option now fails
+ with a clear error message.
+
+:jira:`PXC-546`: Optimized the operation of SST and IST
+ when a node fails during startup.
+
+:jira:`PXC-547`: The ``wsrep_desync`` variable can be enabled only
+ after a node was synced.
```

Comment:
s/node was synced/node is synced with cluster. (In short can't be set during node bootup configuration)/gc